### PR TITLE
Add readme.md

### PR DESCRIPTION
**Pull request by Hirnstromwelle**
Add readme.md
fix naming movieList styling
movieList props of the component as read-only

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Movie-Tracker/Library/Db
+
+## About the Project
+Our Movie/Film Management App is the perfect solution to organize your personal film library and keep track of your favorite movies. With this user-friendly application, you can effortlessly capture, manage, and discover all your movies. Never miss the title of a movie or the names of the lead actors or directors whose works you'd like to watch again. Our app makes it easy to enjoy your passion for movies and maintain a well-organized library. Discover and enjoy movies in a new way - with our Movie App!
+
+## Live Demo
+Visit our app in action on [Render](https://app-5yds.onrender.com)!
+
+## Technologies
+- **Frontend**: TypeScript
```
Comment (mpagels) [inline readme.md:10]: Here I would write: React with Typescript

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- **Backend**: Java
```
Comment (mpagels) [inline readme.md:11]: Here I would write Java Spring & MongoDB

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+## Team Members
+- [Alex](https://github.com/Hirnstromwelle)
+- [Dominic](https://github.com/makitango)
+- [Fabian](https://github.com/FabianMariusLink)
+
+## Getting Started
+
+Run `mvn dependency:resolve` to resolve dependencies with Maven.
+Install all npm dependencies with `npm install`.
+To run the app in development mode, use `npm run dev`, then open http://localhost:3000 to view it in the browser.
```
Comment (mpagels) [inline readme.md:23]: When I run `mvn dependency:resolve` nothing happens on my machine! ;-) 

Recommend adding the `clone the repo` info first ;-)

Comment (mpagels) [reply readme.md:23]: Do I not have to start the backend as well?

Comment (mpagels) [reply readme.md:23]: How do I handle the mongoDB?

**Review by mpagels — APPROVED** [review]
lgtm! Some minor addition ideas for the readme. Good work here

File: readme.md (hunk 3 above)
Comment (Hirnstromwelle) [reply readme.md]: Our suggestion: we delete the text, and you can use the deployment on render.

**Pull request merged**